lib/ace/ext/papeeria/highlighter.coffee

```diff
@@ -38,7 +38,8 @@ define((require, exports, module) ->
         # will return wrong result (e.g. for this text: {\foo}_  it will return { as the result)
         # To fix it we increment columnin the position for searching leftwards.
         positionLeftwards = editor.getCursorPosition()
-        positionLeftwards.column += 1
+        positionLeftwards.column += 1 
```

Comment:
by the way, will it help if we just check if the column is already positioned at the line end and move positionLeftwards to the beginning of the next line if it is?